app/assets/stylesheets/sign_in.scss

```diff
@@ -10,10 +10,10 @@ body#sign-in {
   margin-top: 7rem;
 
   h1 {
-    color: #f5f5f5;
-    letter-spacing: -1.5rem;
-    line-height: 16rem;
-    font-size: 17rem;
+    color: #FFF;
+    opacity: 0.5;
```

Comment:
Properties should be ordered color, font-size, font-weight, letter-spacing, opacity